content/en/blog/_posts/Testing your Kubernetes Cluster - Acceptance and Conformance

```diff
@@ -0,0 +1,226 @@
+Testing the Kubernetes Cluster – Acceptance and Conformance.
```

Comment:
This post needs front matter. For examples, take a look at other files in https://github.com/kubernetes/website/tree/master/content/en/blog/_posts.